clipping_manager/forms.py

```diff
@@ -26,3 +26,33 @@ class UploadTextClippings(forms.Form):
     author = forms.CharField(
         required=False,
     )
+
+
+class BookEmailInclusionForm(forms.Form):
+    """Form to toggle which books are included in email deliveries"""
+    
+    def __init__(self, user, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        from clipping_manager.models import Book
+        
+        # Get user's books that have clippings, sorted by include_in_email (included first) then title
+        books = Book.objects.for_user(user).not_empty().order_by('-include_in_email', 'title')
+        
+        for book in books:
+            field_name = f'book_{book.id}'
+            self.fields[field_name] = forms.BooleanField(
+                label=str(book),
```

Comment:
Maybe rather explicitly use the book's title or whatever helper functions/properties we have on the model instead of the implicit `__str__`?